tests/ci_build/test_mgpu.sh

```diff
@@ -4,5 +4,5 @@ set -e
 cd python-package
 python setup.py install --user
 cd ..
-python -m nose -v --eval-attr='(not slow) and mgpu' tests/python-gpu/
+pytest -s --fulltrace -m "not slow" tests/python-gpu
```

Comment:
```suggestion
pytest -v -s --fulltrace -m "(not slow) and mgpu" tests/python-gpu
```
1. Add `-v` to print out name of all tests
2. Add `and mgpu` to preserve existing behavior